x-pack/solutions/observability/plugins/apm/public/components/alerting/rule_types/register_apm_rule_types.ts

```diff
@@ -49,6 +51,7 @@ export function registerApmRuleTypes(observabilityRuleTypeRegistry: Observabilit
     }),
     requiresAppContext: false,
     defaultActionMessage: errorCountMessage,
+    defaultRecoveryMessage: transactionErrorRateRecoveryMessage,
```

Comment:
shouldn't it be errorCountRecoveryMessage?

Reply:
Nice catch 👍🏻